project/docker/nginx.conf

```diff
@@ -0,0 +1,20 @@
+server {
+        listen 80 default_server;
+        listen [::]:80 default_server;
```

Comment:
You don't need `ipv6only=on` here?

Reply:
In https://github.com/tomeshnet/ipfs-live-streaming/pull/10/files#diff-4c273aedc9d65662c112bb0242307457R12 I had to add it otherwise it fails to bind the ipv6, but that may be before I bump the nginx version, so maybe it was a certain nginx version where this became default.